content/academy/deploying_your_code/output_schema.md

```diff
@@ -0,0 +1,174 @@
+                                "format": "image"
+                            },
+                            "___EXAMPLE_LINK_URL_FIELD___": {
+                                "label": "Clickable link",
+                                "format": "link"
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+To configure the output schema, simply replace the fields in the template with the relevant fields to you actor.
+
+For reference, you can use the [Zappos Scraper source code](https://github.com/PerVillalva/zappos-scraper-actor/blob/main/.actor/actor.json) as an example of how the final implementation of the output tab should look like in a live actor.
+
+```json
+{
+    "actorSpecification": 1,
+    "name": "zappos-scraper",
+    "title": "Zappos Scraper",
+    "description": "",
+    "version": "1.0.0",
+    "storages": {
+        "dataset": {
+            "actorSpecification": 1,
+            "title": "Zappos.com Dataset",
+            "description": "",
+            "views": {
+                "places": {
+                    "title": "Overview",
+                    "description": "It can take about one minute until the first results are available.",
+                    "transformation": {
+                        "fields": [
+                            "imgUrl",
+                            "brand",
+                            "name",
+                            "SKU",
+                            "inStock",
+                            "onSale",
+                            "price",
+                            "url"
+                        ]
+                    },
+                    "display": {
+                        "component": "table",
+                        "properties": {
+                            "imgUrl": {
+                                "label": "Product image",
+                                "format": "image"
+                            },
+                            "url": {
+                                "label": "Link",
+                                "format": "link"
+                            },
+                            "brand": {
+                                "label": "Brand",
+                                "format": "text"
+                            },
+                            "name": {
+                                "label": "Name",
+                                "format": "text"
+                            },
+                            "SKU": {
+                                "label": "SKU",
+                                "format": "text"
+                            },
+                            "inStock": {
+                                "label": "In Stock",
+                                "format": "boolean"
+                            },
+                            "onStock": {
+                                "label": "On Sale",
```

Comment:
`onStock` `On Sale`. One of them probably isn't correct :)

Reply:
Nice catch. It was supposed to be onSale 😄 Fixed it.